src/Xamarin.Android.Build.Tasks/Tasks/AndroidComputeResPaths.cs

```diff
@@ -133,16 +133,52 @@ public override bool RunTask ()
 					Log.LogCodedError ("XA2001", file: item.ItemSpec, lineNumber: 0, message: Properties.Resources.XA2001, item.ItemSpec);
 					continue;
 				}
+				if (intermediateFiles.TryGetValue (dest, out ITaskItem conflict)) {
+					string conflictItemSpec = conflict.GetMetadata ("OriginalItemSpec");
+					string conflictPath = Path.GetFullPath (conflictItemSpec);
+					string path = Path.GetFullPath (item.ItemSpec);
+					if (string.Compare (conflictPath, path, StringComparison.Ordinal) == 0) {
+						Log.LogCodedWarning (
+							"XA1029",
+							file: conflictItemSpec,
+							lineNumber: 0,
+							message: Properties.Resources.XA1029,
+							conflictItemSpec,
+							conflict.GetMetadata ("LogicalName"),
```

Comment:
Is `%(LogicalName)` ever blank here? I think we would have to set it--it doesn't seem like MSBuild would fill this in by default.

Reply:
Ohhh. Now I remember why I did this. It's because this warning will always happen on an item that has already been through the full loop, so the following code has necessarily run previously for the `conflict` object:

https://github.com/xamarin/xamarin-android/blob/f1a6e22ffa72655b96766da7ee1d1ce0ae2f5008/src/Xamarin.Android.Build.Tasks/Tasks/AndroidComputeResPaths.cs#L137

So `LogicalName` will always contain the value of `rel` for each item, where `rel` has the carefully set up non-empty value as prepared at the beginning of the method.